### DISCUSSION TITLE
using xtb-iff combing gbsa

### DISCUSSION THREAD

**Discussion by boyuezhong** [grimme-lab/xtb#781] [General]
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is.

**Describe the solution you'd like**

**Describe alternatives you've considered**

**If possible state how you can assist in providing data or code to to implement the feature**
A short description of what you can provide to make the feature possible.

**Additional context**
Add any other context about the feature request here.

**Comment by boyuezhong** [comment]
How to combine xtb-iff with gbsa algorithm

**Comment by cplett** [comment]
Hi, if you mean the outdated `xtbiff` binary (https://github.com/grimme-lab/xtbiff), it is not possible to combine this with an implicit solvent model.
However, the new and preferred [`aISS` docking algorithm](https://xtb-docs.readthedocs.io/en/latest/xtb_docking.html) implemented in `xtb` can be invoked with `--gbsa <solvent>` or `--alpb <solvent>`. A list of available solvents can be found [here](https://xtb-docs.readthedocs.io/en/latest/gbsa.html).

**Reply by boyuezhong** [reply]
The old version of xtb-iff can calculate the single point energy, but solvation cannot be considered. The new aISS docking algorithm can consider solvation, but it seems unable to calculate the single point energy. I want the new aISS docking algorithm not to search for conformation, but to score the complex. However, I can't find the right way.

**Comment by boyuezhong** [comment]
How does  aiss calculate the single-point interaction energy

**Comment by cplett** [comment]
The single-point energies for the structure screening are calculated with the xtb-IFF energy and the interaction energies of the final structures are computed with the chosen method of geometry optimizations (e.g. GFN2-xTB, GFN-FF or GFN-xTB). Be careful, that these final interaction energies are computed based on the geometries of your input. Thus, you should provide already optimized input geometries.
